app/models/github_repo.rb

```diff
@@ -27,31 +27,8 @@ def self.upsert(user, **params)
   end
 
   def self.update_to_latest
-    where("updated_at < ?", 26.hours.ago).includes(:user).find_each do |repo|
-      user = repo.user
-      next unless user
```

Comment:
Removed bc we are now enforcing(thanks @rhymes!)  that when a user is removed all it's GithubRepos are destroyed. I cleaned up the orphaned ones in the database.